src/main/resources/assets/js/components/TaskListComponent.jsx

```diff
@@ -71,16 +81,7 @@ define([
               </td>
             </tr>
           </tbody>;
-      } else if (this.props.fetchState === States.STATE_ERROR) {
-        taskNodes =
-          <tbody>
-            <tr>
-              <td className="text-center text-danger" colSpan="7">
-                Error fetching tasks. Refresh the list to try again.
-              </td>
-            </tr>
-          </tbody>;
-      } else if (tasksLength === 0) {
+      } else if (tasksLength === 0 && !fetchError) {
```

Comment:
Can you create a variable here like it is done in TaskListDetail to tell if there's an error?
```
var hasError =
    this.props.fetchState === States.STATE_ERROR ||
    tasksLength === 0;
```